cpp/subprojects/orc.wrap

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[wrap-file]
+source_url = https://www.apache.org/dyn/closer.lua/orc/orc-1.9.6/orc-1.9.6.tar.gz?action=download
+source_fallback_url = https://dlcdn.apache.org/orc/orc-1.9.6/orc-1.9.6.tar.gz
+source_filename = orc-1.9.6.tar.gz
```

Comment:
I had to intentionally limit the version of orc < 2, as in 2.0 they introduced a new externalproject called `orc-format` that contains just the proto files for orc. Meson wrapping a CMake configuration to call an ExternalProject that itself is not a CMake project seemed pretty fragile...